test/other/debug.aws

```diff
@@ -1,7 +1,8 @@
 [default]
 something_other=sdfsdf
-aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+aws_access_key_id =      AKIAIOSFODNN7EXAMPLE
```

Comment:
No that's on purpose: the three parameters are using very different spaces in order to test that the detection works properly :)